public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-admin.php

```diff
@@ -773,14 +774,17 @@ public static function notify_new_wordcamp_in_slack( $wordcamp ) {
 			}
 
 			// Not translating any string because they will be sent to slack.
-			$city         = get_post_meta( $wordcamp->ID, 'Location', true );
 			$start_date   = get_post_meta( $wordcamp->ID, 'Start Date (YYYY-mm-dd)', true );
 			$wordcamp_url = get_post_meta( $wordcamp->ID, 'URL', true );
-			$title        = 'New WordCamp scheduled!!!';
+			$url          = wp_parse_url( filter_var( $wordcamp_url, FILTER_VALIDATE_URL ) );
+			$tld          = get_top_level_domain();
+			$is_event     = "events.wordpress.$tld" === $url['host'];
```

Comment:
It might be nice to turn this into a helper function where you could pass in a URL and get back `wordcamp` or `next-gen`. That'd be a good compliment to `is_wordcamp_type()`.

We can always do that in future though, not a big deal if you don't want to for this PR.

Reply:
How about this: https://github.com/WordPress/wordcamp.org/pull/1066/commits/b158437a14ccf3b05f11345ff3dc07a0f5f3ab1d?

I chose to keep the language specific to events because i don't think we need expose the type slugs in the plugins.

Reply:
That looks good 👍🏻